docs/docs/recipes/protect-your-api/README.md

```diff
@@ -7,21 +7,21 @@ import TabItem from '@theme/TabItem';
 
 # ⚔️ Protect your API
 
-Along with a fluent user sign-in experience, by default, Logto comes with the [API resources](../../references/resources/README.md) authorization service. It will help you protect the private APIs resources from anonymous identities. Let's walk through the following steps and implement the API resource authorization guard of your own using Logto.
+With a fluent user sign-in experience, by default, Logto comes with the [API resources](../../references/resources/README.md) authorization service. It will help you protect the your APIs resources from anonymous identities. Let's walk through the following steps and implement the API resource authorization guard of your own using Logto.
 
 ## Register the API resource through Logto **Admin Console**
 
 Logto will identify the registered [API resources](../../references/resources/README.md) from an authorization request and issue an audience-restricted `access_token` accordingly.
```

Comment:
Maybe not accurate in Logto.
```
Logto will recognize the registered API resources when provided in an authorization request and issue an audience-restricted `access_token` when requested with the specific resource.
```

Reply:
` when provided` feels redundant to me though.